submission/persistence-filesystem/src/test/kotlin/ac/uk/ebi/biostd/persistence/filesystem/fire/FireFilesServiceTest.kt

```diff
@@ -50,23 +50,26 @@ internal class FireFilesServiceTest(
             fireClient.findByMd5("the-md5")
         }
         verify(exactly = 0) {
-            fireClient.downloadByFireId("the-fire-id", "file.txt")
-            fireClient.setPath("the-fire-id", "/S-BSST/001/S-BSST1/Files/folder/file.txt")
-            fireClient.save(any(), "the-md5", 123L)
+            fireClient.downloadByFireId(any(), any())
```

Comment:
Considering we have control over the whole scenario, I don't see the need for using `any()`

Reply:
Imagine one accidentaly set the wrong value in this `verify(0)` then test will pass and we may think method one expecting to not be call was not call but indeed it was call but with diferent parameters.  IMO using any in positive verify (greater than 1) is an antipattern and using in `verify(0)` is safer